spec/factories/bulk_upload_error.rb

```diff
@@ -9,7 +9,7 @@
     tenant_code { SecureRandom.hex(4) }
     property_ref { SecureRandom.hex(4) }
     purchaser_code { SecureRandom.hex(4) }
-    field { "field_#{rand(134)}" }
+    field { "field_#{rand(1..134)}" }
```

Comment:
`rand(134)` generates a number in the range 0-133. We want a number in the range 1-134.